content/en/docs/instrumentation/python/exporters.md

```diff
@@ -136,71 +135,28 @@ OTLPSpanExporter(endpoint="<traces-endpoint>/v1/traces")
 
 There is not currently an OTLP/HTTP metric exporter.
 
-## Jaeger
+### Jaeger
 
 If you are using [Jaeger](https://www.jaegertracing.io/) to visualize trace
-data, you'll need to set it up first. This is how to run it in a docker
-container:
+data, This is how to run it in a docker container:
 
-```console
-$ docker run -d --name jaeger \
+```shell
+docker run -d --name jaeger \
   -e COLLECTOR_ZIPKIN_HOST_PORT=:9411 \
-  -p 5775:5775/udp \
+  -e COLLECTOR_OTLP_ENABLED=true \
   -p 6831:6831/udp \
   -p 6832:6832/udp \
   -p 5778:5778 \
   -p 16686:16686 \
-  -p 14268:14268 \
+  -p 4317:4317 \
+  -p 4318:4318 \
   -p 14250:14250 \
+  -p 14268:14268 \
+  -p 14269:14269 \
   -p 9411:9411 \
   jaegertracing/all-in-one:latest
 ```
 
-Next, install the Jaeger exporter package:
-
-```console
-$ pip install opentelemetry-exporter-jaeger
-```
-
-This will install packages for both:
-
-* `opentelemetry-exporter-jaeger-thrift`
-* `opentelemetry-exporter-jaeger-proto-grpc`
-
-You can use either to export your traces to Jaeger.
-
-Once the package is installed, you can configure the exporter when initializing tracing:
-
-```python
-from opentelemetry import trace
-from opentelemetry.exporter.jaeger.thrift import JaegerExporter
-from opentelemetry.sdk.resources import SERVICE_NAME, Resource
-from opentelemetry.sdk.trace import TracerProvider
-from opentelemetry.sdk.trace.export import BatchSpanProcessor
-
-resource = Resource(attributes={
-    SERVICE_NAME: "your-service-name"
-})
-
-jaeger_exporter = JaegerExporter(
-    agent_host_name="localhost",
-    agent_port=6831,
-)
```

Comment:
Why are we removing this altogether? While the newer version of jaeger support OTLP directly, this is still important for users who would want to use other protocol such as thrift/UDP or thrift/HTTP. I would keep this and add a new note that jaeger now supports OTLP directly and one can just use OTLP exporter to send data directly.

Reply:
@srikanthccv we felt that just moving to OTLP for documenting was adequate as per here: https://github.com/open-telemetry/opentelemetry.io/issues/1567 -- I don't personally believe that the intersection of people who need to know how to send data to Jaeger _and_ want to explicitly use the thrift protocol is high enough to justify adding two ways to do things to docs. I don't have any data to back that up, just that in general non-defaults like this drop off like a cliff in in terms of usage.

Reply:
Many probably prefer using OTLP directly with Jaeger. I am not against documenting that. My point is why do we need to remove the existing documentation which shows how to send using thrift to jaeger to convey that. This PR is getting rid of jaeger exporter section totally and adding a sub section under OTLP. If we accept this change where do we send the people who come looking for thrift documentation (for whatever reason they might have)? I view this page as a source of guide for end user rather than recommendation page backed by data.

Reply:
So I'm of two minds for this:

* I agree in principle that having reference documentation for edge cases like this matters
* I can't think of a good reason why someone, outside of using an older version of jaeger, would need or want to use thrift rather than OTLP, and I'd much rather encourage them to switch to OTLP. But I am not used to operating environments that use thrift so I don't know how realistic that stance is

FWIW there are docs for the exporter itself here: https://opentelemetry-python.readthedocs.io/en/latest/exporter/jaeger/jaeger.html

So I'm fine with leaving the Python docs as-is, or at least linking to this, for a time.

@svrnm what do you think?

Reply:
> I can't think of a good reason why someone, outside of using an older version of jaeger, would need or want to use thrift rather than OTLP, and I'd much rather encourage them to switch to OTLP. But I am not used to operating environments that use thrift so I don't know how realistic that stance is

In my experience, It's not the version but rather the deployment pattern. In cases where jaeger runs as an agent thrift/UDP is preferred and where it's not possible to run the agent alongside then other ways are preferred. The thrift exporter on PyPI gets significant enough downloads each month https://pypistats.org/packages/opentelemetry-exporter-jaeger-thrift. I want to reiterate my concern. I am fine with actively encouraging people to move to direct OTLP but that shouldn't mean we remove the documentation for thrift/UDP or thrift/HTTP exporters.

>FWIW there are docs for the exporter itself here: https://opentelemetry-python.readthedocs.io/en/latest/exporter/jaeger/jaeger.html

We have been in the process of getting rid of them and asking users to consider opentelemetry.io docs as one true source and only use the python readthedocs.io for the component API documentation.

Reply:
@Gaurang-Patel can you revert the changes to Python for now? I think we can figure out a good way forward, but let's unblock .NET for now, which is approved.

Reply:
@cartermp I'll do it in a day or two.